### ISSUE TITLE
InterstitialScreen: accessibility testing issue set 2

### ISSUE THREAD

**Issue by amal-k-joy** [carbon-design-system/ibm-products#7461]
These are the set 2 of issues raised by Raghu on his accessibility testing.

Issue   title/Summary | Issue Description | Recommended Fix | User Impact | Developer comments
-- | -- | -- | -- | --
Non Interactive element Modal Div receiving  Tab focus . | Modal Div   receiving  Tab focus .:     Tab focus on Non interactive controls    such as Text , Div etc.  will   make  the Keyboard /Screen reader users   confuse. | Avoid Tab focus   from Non Interactive Elements such as    Div , Text etc. | When   noninteractive elements like texts, graphics or icons receive tab focus,   screen reader users can wrongly assume them to be interactive controls and   can even try to activate them. |  
Content modification on the Modal due to activation on Next   /Previous Buttons are not conveyed  to   screen reader users. | Content   modification on the Modal due to activation on Next /Previous Buttons are not   conveyed  to screen reader users.     By clicking  Next /Previous  content in the Modal got updated - However   , screen reader   users does not get   any clue on this content modifications. | Make   the dynamic  content updates  indicated to screen reader users  by using appropriate technics :      For example : Providing  a  status indication  for screen reader users  such as - "Step 3    Provide your experience  Details " Step 4 - "Provide   your  Qualificational  information " etc.      while moving to these steps - screen reader should auto announce the   Topic  of the Steps | If   the screen reader fails to identify a dynamic content change, it affects   users who are blind as they may not know about the change in the page. |  
Interactive components with in the Modal not reachable  by Tab key    consistently . | Modal   controls(step1) are not reachable by Tab consistently - also  once the    Elements receives focus - Tab order not following the visual   order  of the  components , which  how these are placed in the  Modal. | Make   the  Modal controls  reachable by Keyboard alone , also Tab   order expected to follow the visual order of the control  consistently. | When   the standard keyboard navigation method is not supported, it impacts the   user’s ability to interact with page controls. This defect affects screen   reader and keyboard-only users. |  
Screen reader announced the content  which is visually available | Based   on the Stepper selection by clicking    Next /Previous Button - Users are able to see  the Content Block related to  that particular Step .     For example : at present  the Step   2  content is visible   , means Step 1, and Step 3  content will be  in visible .     However , regardless  of the Stepper   selection all the Time screen reader    announcing  all the Step 1, 2,   & 3  Content sections  all the Time .     Here - screen reader announcing the content visually not  presented - which confuse  screen reader users. | Screen   reader not expected to announce the    content which is not visually presented - avoid the same.     If the step 2  content is  currently presented , screen Reder expected   to   announce only  the step 2    content , not step 1  ,  or step 3.     Note :      Hide all the content which is not visually    available , from screen reader users to avoid confusions. | If   the screen reader identifies the visually hidden content, it tends to mislead   users who are blind. |
